modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -27,22 +27,28 @@ apiVersion: monitoring.openshift.io/v1
 kind: AlertRelabelConfig
 metadata:
   name: watchdog
-  namespace: openshift-monitoring
+  namespace: openshift-monitoring # <1>
 spec:
   configs:
-  - sourceLabels: [alertname,severity] <1>
-    regex: "Watchdog;none" <2>
-    targetLabel: severity <3>
-    replacement: critical <4>
-    action: Replace <5>
+  - sourceLabels: [alertname,severity] # <2>
+    regex: "Watchdog;none" # <3>
+    targetLabel: severity # <4>
+    replacement: critical # <5>
+    action: Replace # <6>
 ----
-<1> The source labels for the values you want to modify.
-<2> The regular expression against which the value of `sourceLabels` is matched.
-<3> The target label of the value you want to modify.
-<4> The new value to replace the target label.
-<5> The relabel action that replaces the old value based on regex matching.
+<1> Ensure that the namespace is `openshift-monitoring`.
+<2> The source labels for the values you want to modify.
+<3> The regular expression against which the value of `sourceLabels` is matched.
+<4> The target label of the value you want to modify.
+<5> The new value to replace the target label.
+<6> The relabel action that replaces the old value based on regex matching.
 The default action is `Replace`.
 Other possible values are `Keep`, `Drop`, `HashMod`, `LabelMap`, `LabelDrop`, and `LabelKeep`.
++
+[IMPORTANT]
+====
+You must create the `AlertRelabelConfig` object in the `openshift-monitoring` namespace. Otherwise, the alert label will not change.
```

Comment:
Please see the [IBM Style guidance](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) on verb tense and avoiding future tense, especially in cause and effect sentences. 

That said, this is not a blocker, so I will go ahead and merge. Feel free to create an issue if you want to track fixing this in another PR or please fix it the next time you are in this doc set.

Reply:
Oops, I saw that @ousleyp recommended using future tense. :) Nevermind the nit, as I hadn't looked at the resolved comments. Sorry for the noise.

Reply:
I almost never advocate for future tense, but it seemed right to me here. I am open to being corrected, though! :)